### ISSUE TITLE
Invalid memory address during CQ loop

### ISSUE THREAD

**Issue by otoolep** [influxdb/influxdb#2352]
This happened during CI testing.

```
	server_integration_test.go:1359: Running test 142 '3-node server failover integration:show continuous queries'
	server_integration_test.go:312: Running 3-node server failover integration:rawDataReturnsInOrder against 2-node cluster
	server_integration_test.go:160: Test: 3-node server failover integration: creating database mydb
	server_integration_test.go:167: Creating retention policy myrp for database mydb
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x5332b2]

goroutine 4286 [running]:
net/http.(*Client).doFollowingRedirects(0x0, 0xc20911b520, 0xb134d0, 0x0, 0x0, 0x0)
	/home/ubuntu/.gvm/gos/go1.4/src/net/http/client.go:304 +0x62
net/http.(*Client).Post(0x0, 0xc2093ea540, 0x31, 0xa564f0, 0x18, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/ubuntu/.gvm/gos/go1.4/src/net/http/client.go:441 +0x10d
github.com/influxdb/influxdb.(*Broker).requestContinuousQueryProcessing(0xc20819e370, 0x9e38b0, 0x4, 0x0, 0x0, 0x0, 0xc20930aa87, 0xa, 0xa78070, 0x20, ...)
	/home/ubuntu/influxdb-build/src/github.com/influxdb/influxdb/broker.go:137 +0x127
github.com/influxdb/influxdb.(*Broker).runContinuousQueries(0xc20819e370)
	/home/ubuntu/influxdb-build/src/github.com/influxdb/influxdb/broker.go:120 +0x3b4
github.com/influxdb/influxdb.(*Broker).continuousQueryLoop(0xc20819e370, 0xc2083ac360)
	/home/ubuntu/influxdb-build/src/github.com/influxdb/influxdb/broker.go:90 +0x56
created by github.com/influxdb/influxdb.(*Broker).RunContinuousQueryLoop
	/home/ubuntu/influxdb-build/src/github.com/influxdb/influxdb/broker.go:63 +0xbb
```

**Comment by otoolep** [comment]
@jwilder

**Comment by jwilder** [comment]
Was fixed via ca845ad1
